android/gradle.properties

```diff
@@ -21,11 +21,6 @@
 # Android operating system, and which are packaged with your app's APK
 # https://developer.android.com/topic/libraries/support-library/androidx-rn
 android.useAndroidX=true
-# Automatically convert third-party libraries to use AndroidX
-android.enableJetifier=true
```

Comment:
no need anymore?

Reply:
It has been removed from the template and it works fine in my first tests. But I will be especially careful this week to see if this kind of changes have an impact.

Reply:
What is the oldest targeted Android version with this new RN upgrade? 

I'm pretty sure that this kind of bug with AndroidX will be easy to detect because we'll have compilations errors and else.

Reply:
What do you mean by oldest targeted Android version?

Current Android SDK versions :
minSdkVersion = 24
compileSdkVersion = 35
targetSdkVersion = 35